worker/crates/action-universal/src/xml/formatter.rs

```diff
@@ -0,0 +1,169 @@
+use reearth_flow_xml::{
+    parser::read_xml,
+    traits::{Element, Node},
+};
+use serde::{Deserialize, Serialize};
+
+use reearth_flow_action::{
+    error::Error, ActionContext, ActionDataframe, ActionResult, ActionValue, AsyncAction,
+    DEFAULT_PORT,
+};
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct XmlFormatter {
+    attribute: String,
+}
+
+#[async_trait::async_trait]
+#[typetag::serde(name = "XMLFormatter")]
+impl AsyncAction for XmlFormatter {
+    async fn run(&self, _ctx: ActionContext, inputs: Option<ActionDataframe>) -> ActionResult {
+        let inputs = inputs.ok_or(Error::input("no inputs"))?;
+        let default = inputs
+            .get(&DEFAULT_PORT)
+            .ok_or(Error::input("no default"))?
+            .clone();
+        let new_default = match default {
+            Some(ActionValue::Array(xs)) => Ok(ActionValue::Array(
+                xs.into_iter()
+                    .map(|x| match x {
+                        ActionValue::Map(mut kv) => {
+                            if let Some(ActionValue::String(src)) = kv.get_mut(&self.attribute) {
+                                *src = match {
+                                    || {
+                                        read_xml(src)?
+                                            .first_child()
+                                            .ok_or(reearth_flow_xml::error::Error::WrongDocument)?
+                                            .to_xml()
+                                    }
+                                }() {
+                                    Ok(x) => x,
+                                    Err(_) => src.to_string(),
```

Comment:
In case of this error, please write in the log
reference: 
https://github.com/reearth/reearth-flow/blob/main/worker/crates/action-universal/src/xml/fragmenter.rs#L193-L197

Reply:
Do we need to let it fail whole when there is an error? Or do we just have to log?

Reply:
@kirisaki 
> Do we need to let it fail whole when there is an error? Or do we just have to log?

Just log it once and you are good to go!
Actually, it needs to be routed to a rejected port, but we'll deal with that later.

Reply:
How do I set parameters `span` and `logger` in `action_log!()`?

Reply:
Pass what is in the ActionContext as is.

Reply:
I pushed changes. Please check it.